OpenRA.Game/Network/OrderManager.cs

```diff
@@ -113,12 +121,17 @@ public void TickImmediate()
 				(clientId, packet) =>
 				{
 					var frame = BitConverter.ToInt32(packet, 0);
+
+					// Packet [int32 frame number][0xBF], sent by server when client dropped
 					if (packet.Length == 5 && packet[4] == 0xBF)
 						frameData.ClientQuit(clientId, frame);
+
 					else if (packet.Length >= 5 && packet[4] == 0x65)
 						CheckSync(packet);
+
 					else if (frame == 0)
 						immediatePackets.Add(Pair.New(clientId, packet));
+
```

Comment:
This is a StyleCop violation which makes our CI fail: [SA1510] An else, catch, or finally statement must not be preceded by a blank line.